### PR TITLE
Fix: ClamAV fails to complete scan (#6022)

### LINKED ISSUE #6022: ClamAV fails to complete scan
Since the upgrade of ClamAV from `1.2.1-27` to `1.3.0-37` on Feb 27th (for the upgrade dependencies ticket #5926, PR #5969), Clamscan have been failing on all deployments with the update.

Note: The ClamAV schedule was updated as part of a separate issue (PR #5943) on Feb 29th, however as seen in the logs below, the failures started after the version update and before the schedule update was applied.

[CloudWatch logs on dev](https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#logsV2:logs-insights$3FqueryDetail$3D~(end~'2024-03-06T06*3a08*3a31.562Z~start~'2024-02-17T18*3a43*3a12.762Z~timeType~'ABSOLUTE~tz~'LOCAL~editorString~'fields*20*40timestamp*2c*20*40message*0a*7c*20filter*20*40logStream*20*3d*20*27*2fvar*2flog*2fmessages*27*0a*7c*20filter*20*40message*20like*20*2fclamscan*20*28failed*7csucceeded*29*2f*20or*20*40message*20like*20*27ClamAV*20update*20process*20started*27*0a*7c*20sort*20*40timestamp*20asc*0a*7c*20limit*20500~queryId~'58767287b31b48d-759b28a-4c628bf-bc17a9c-e439378acf173c550e752b~source~(~'*2faws*2fcwagent*2fazul-gitlab))):

Screenshot 2024-03-08 at 5 43 10 PM

[CloudWatch logs for a failing run on dev](https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#logsV2:logs-insights$3FqueryDetail$3D~(end~'2024-02-29T10*3a16*3a32.125Z~start~'2024-02-28T05*3a45*3a07.086Z~timeType~'ABSOLUTE~tz~'LOCAL~editorString~'fields*20*40timestamp*2c*20*40message*0a*7c*20filter*20*40logStream*20*3d*20*27*2fvar*2flog*2fmessages*27*0a*7c*20filter*20*40message*20like*20*27clam*27*0a*7c*20sort*20*40timestamp*20asc*0a*7c*20limit*20500~queryId~'58767287b31b48d-759b28a-4c628bf-bc17a9c-e439378acf173c550e752b~source~(~'*2faws*2fcwagent*2fazul-gitlab))):
```
Feb 28 17:58:08 ip-172-71-0-215 docker: clamscan: ClamAV update process started at Wed Feb 28 17:58:08 2024
Feb 28 17:58:08 ip-172-71-0-215 docker: clamscan: daily database available for update (local version: 27195, remote version: 27199)
Feb 28 17:58:11 ip-172-71-0-215 docker: clamscan: Testing database: '/var/lib/clamav/tmp.86372efe5b/clamav-f02af4d30b13f416548f636de1feed88.tmp-daily.cld' ...
Feb 28 17:58:23 ip-172-71-0-215 docker: clamscan: Database test passed.
Feb 28 17:58:23 ip-172-71-0-215 docker: clamscan: daily.cld updated (version: 27199, sigs: 2054066, f-level: 90, builder: raynman)
Feb 28 17:58:23 ip-172-71-0-215 docker: clamscan: main.cvd database is up-to-date (version: 62, sigs: 6647427, f-level: 90, builder: sigmgr)
Feb 28 17:58:23 ip-172-71-0-215 docker: clamscan: bytecode database available for update (local version: 334, remote version: 335)
Feb 28 17:58:23 ip-172-71-0-215 docker: clamscan: Testing database: '/var/lib/clamav/tmp.86372efe5b/clamav-ddb5190ef1bc551a22ec5b94c0d8c61c.tmp-bytecode.cld' ...
Feb 28 17:58:23 ip-172-71-0-215 docker: clamscan: Database test passed.
Feb 28 17:58:23 ip-172-71-0-215 docker: clamscan: bytecode.cld updated (version: 335, sigs: 86, f-level: 90, builder: raynman)
Feb 28 17:58:23 ip-172-71-0-215 docker: clamscan: WARNING: Clamd was NOT notified: Can't connect to clamd through /tmp/clamd.sock: No such file or directory
Feb 28 17:58:23 ip-172-71-0-215 docker: clamscan: freshclam succeeded
Feb 28 18:27:21 ip-172-71-0-215 docker: clamscan: LibClamAV Warning: cli_scanxz: decompress file size exceeds limits - only scanning 105906176 bytes
Feb 28 18:27:55 ip-172-71-0-215 docker: clamscan: LibClamAV Warning: cli_scanxz: decompress file size exceeds limits - only scanning 105906176 bytes
Feb 28 18:28:03 ip-172-71-0-215 docker: clamscan: LibClamAV Warning: cli_scanxz: decompress file size exceeds limits - only scanning 105906176 bytes
Feb 28 18:28:09 ip-172-71-0-215 docker: clamscan: LibClamAV Warning: cli_scanxz: decompress file size exceeds limits - only scanning 105906176 bytes
Feb 28 18:28:16 ip-172-71-0-215 docker: clamscan: LibClamAV Warning: cli_scanxz: decompress file size exceeds limits - only scanning 105906176 bytes
Feb 28 18:28:22 ip-172-71-0-215 docker: clamscan: LibClamAV Warning: cli_scanxz: decompress file size exceeds limits - only scanning 105906176 bytes
Feb 28 18:29:11 ip-172-71-0-215 docker: clamscan: LibClamAV Warning: cli_scanxz: decompress file size exceeds limits - only scanning 105906176 bytes
Feb 28 18:29:21 ip-172-71-0-215 docker: clamscan: LibClamAV Warning: cli_scanxz: decompress file size exceeds limits - only scanning 105906176 bytes
Feb 28 18:29:27 ip-172-71-0-215 docker: clamscan: LibClamAV Warning: cli_scanxz: decompress file size exceeds limits - only scanning 105906176 bytes
Feb 28 18:29:32 ip-172-71-0-215 docker: clamscan: LibClamAV Warning: cli_scanxz: decompress file size exceeds limits - only scanning 105906176 bytes
Feb 28 18:29:39 ip-172-71-0-215 docker: clamscan: LibClamAV Warning: cli_scanxz: decompress file size exceeds limits - only scanning 105906176 bytes
Feb 28 18:29:45 ip-172-71-0-215 docker: clamscan: LibClamAV Warning: cli_scanxz: decompress file size exceeds limits - only scanning 105906176 bytes
Feb 28 20:15:13 ip-172-71-0-215 docker: clamscan: LibClamAV Error: cli_html_normalise: style chunk size underflow
Feb 28 21:39:18 ip-172-71-0-215 docker: clamscan: LibClamAV Error: cli_html_normalise: style chunk size underflow
Feb 28 22:08:40 ip-172-71-0-215 docker: clamscan: LibClamAV Error: cli_html_normalise: style chunk size underflow
Feb 28 23:38:14 ip-172-71-0-215 kernel: clamscan[28473]: segfault at 7fe7e1319372 ip 00007fe8345a4fde sp 00007ffdf4a03df0 error 4 in libclamav.so.12.0.2[7fe83441c000+5c1000]
Feb 28 23:38:15 ip-172-71-0-215 docker: clamscan: clamscan failed
Feb 28 23:38:15 ip-172-71-0-215 docker: clamscan: Segmentation fault (core dumped)
Feb 28 23:38:24 ip-172-71-0-215 systemd: clamscan.service: main process exited, code=exited, status=1/FAILURE
Feb 28 23:38:24 ip-172-71-0-215 systemd: Unit clamscan.service entered failed state.
Feb 28 23:38:24 ip-172-71-0-215 systemd: clamscan.service failed.
```

For reproduction, see [comment](https://github.com/DataBiosphere/azul/issues/6022#issuecomment-2006172997).

### PR DESCRIPTION
**Pull request by dsotirho-ucsc**
Connected issues: #6022

### Author

- [x] PR is a draft
- [x] Target branch is `develop`
- [x] Name of PR branch matches `issues/<GitHub handle of author>/<issue#>-<slug>`
- [x] On ZenHub, PR is connected to all issues it (partially) resolves
- [x] PR description links to connected issues
- [x] PR title matches<sup>1</sup> that of a connected issue <sub>or comment in PR explains why they're different</sub>
- [x] PR title references all connected issues
- [x] For each connected issue, there is at least one commit whose title references that issue

### Author (partiality)

- [x] Added `p` tag to titles of partial commits
- [x] This PR is labeled `partial` <sub>or completely resolves all connected issues</sub>
- [x] This PR partially resolves each of the connected issues <sub>or does not have the `partial` label</sub>

<sup>1</sup> when the issue title describes a problem, the corresponding PR
title is `Fix: ` followed by the issue title

### Author (chains)

- [x] This PR is blocked by previous PR in the chain <sub>or is not chained to another PR</sub>
- [x] The blocking PR is labeled `base` <sub>or this PR is not chained to another PR</sub>
- [x] This PR is labeled `chained` <sub>or is not chained to another PR</sub>

### Author (reindex, API changes)

- [x] Added `r` tag to commit title <sub>or this PR does not require reindexing</sub>
- [x] This PR is labeled `reindex:dev` <sub>or does not require reindexing `dev`</sub>
- [x] This PR is labeled `reindex:anvildev` <sub>or does not require reindexing `anvildev`</sub>
- [x] This PR is labeled `reindex:anvilprod` <sub>or does not require reindexing `anvilprod`</sub>
- [x] This PR is labeled `reindex:partial` and its description documents the specific reindexing procedure for `dev`, `anvildev`, `anvilprod` and `prod` <sub>or requires a full reindex or carries none of the labels `reindex:dev`, `reindex:anvildev`, `reindex:anvilprod` and `reindex:prod`</sub>
- [x] This PR and its connected issues are labeled `API` <sub>or this PR does not modify a REST API</sub>
- [x] Added `a` (`A`) tag to commit title for backwards (in)compatible changes <sub>or this PR does not modify a REST API</sub>
- [x] Updated REST API version number in `app.py` <sub>or this PR does not modify a REST API</sub>

### Author (upgrading deployments)

- [x] Documented upgrading of deployments in UPGRADING.rst <sub>or this PR does not require upgrading deployments</sub>
- [x] Added `u` tag to commit title <sub>or this PR does not require upgrading deployments</sub>
- [x] Ran `make image_manifests.json` and committed the resulting changes <sub>or this PR does not modify `azul_docker_images`, or any other variables referenced in the definition of that variable</sub>
- [x] This PR is labeled `upgrade` <sub>or does not require upgrading deployments</sub>
- [x] This PR is labeled `deploy:shared` <sub>or does not modify `image_manifests.json`, and does not require deploying the `shared` component for any other reason</sub>
- [x] This PR is labeled `deploy:gitlab` <sub>or does not require deploying the `gitlab` component</sub>
- [x] This PR is labeled `deploy:runner` <sub>or does not require deploying the `runner` image</sub>

### Author (operator tasks)

- [x] Added checklist items for additional operator tasks <sub>or this PR does not require additional tasks</sub>

### Author (hotfixes)

- [x] Added `F` tag to main commit title <sub>or this PR does not include permanent fix for a temporary hotfix</sub>
- [x] Reverted the temporary hotfixes for any connected issues <sub>or the `prod` branch has no temporary hotfixes for any connected issues</sub>

### Author (before every review)

- [x] Rebased PR branch on `develop`, squashed old fixups
- [x] Ran `make requirements_update` <sub>or this PR does not modify `requirements*.txt`, `common.mk`, `Makefile` and `Dockerfile`</sub>
- [x] Added `R` tag to commit title <sub>or this PR does not modify `requirements*.txt`</sub>
- [x] This PR is labeled `reqs` <sub>or does not modify `requirements*.txt`</sub>
- [x] `make integration_test` passes in personal deployment <sub>or this PR does not modify functionality that could affect the IT outcome</sub>

### Peer reviewer (after requesting changes)

Uncheck the *Author (before every review)* checklists.

### Peer reviewer (after approval)

- [x] PR is not a draft
- [x] Ticket is in *Review requested* column
- [x] Requested review from system administrator
- [x] PR is assigned to only the system administrator

### System administrator (after requesting changes)

Uncheck the *before every review* checklists. Update the `N reviews` label.

### System administrator (after approval)

- [x] Actually approved the PR
- [x] Labeled connected issues as `demo` or `no demo`
- [x] Commented on connected issues about demo expectations <sub>or all connected issues are labeled `no demo`</sub>
- [x] Decided if PR can be labeled `no sandbox`
- [x] A comment to this PR details the completed security design review <sub>or this PR is a promotion or a backport</sub>
- [x] PR title is appropriate as title of merge commit
- [x] `N reviews` label is accurate
- [x] Moved ticket to *Approved* column
- [x] PR is assigned to only the operator

### Operator (before pushing merge the commit)

- [x] Checked `reindex:…` labels and `r` commit title tag
- [x] Checked that demo expectations are clear <sub>or all connected issues are labeled `no demo`</sub>
- [x] PR has checklist items for upgrading instructions <sub>or PR is not labeled `upgrade`</sub>
- [x] Squashed PR branch and rebased onto `develop`
- [x] Sanity-checked history
- [x] Pushed PR branch to GitHub
- [x] Ran `_select dev.shared && CI_COMMIT_REF_NAME=develop make -C terraform/shared apply_keep_unused` <sub>or this PR is not labeled `deploy:shared`</sub>
- [x] Ran `_select dev.gitlab && CI_COMMIT_REF_NAME=develop make -C terraform/gitlab apply` <sub>or this PR is not labeled `deploy:gitlab`</sub>
- [x] Ran `_select anvildev.shared && CI_COMMIT_REF_NAME=develop make -C terraform/shared apply_keep_unused` <sub>or this PR is not labeled `deploy:shared`</sub>
- [x] Ran `_select anvildev.gitlab && CI_COMMIT_REF_NAME=develop make -C terraform/gitlab apply` <sub>or this PR is not labeled `deploy:gitlab`</sub>
- [x] Ran `_select anvilprod.shared && CI_COMMIT_REF_NAME=develop make -C terraform/shared apply_keep_unused` <sub>or this PR is not labeled `deploy:shared`</sub>
- [x] Ran `_select anvilprod.gitlab && CI_COMMIT_REF_NAME=develop make -C terraform/gitlab apply` <sub>or this PR is not labeled `deploy:gitlab`</sub>
- [x] Checked the items in the next section <sub>or this PR is labeled `deploy:gitlab`</sub>
- [x] Assigned system administrator <sub>or this PR is not labeled `deploy:gitlab`</sub>

### System administrator

- [x] Background migrations for `dev.gitlab` are complete <sub>or this PR is not labeled `deploy:gitlab`</sub>
- [x] Background migrations for `anvildev.gitlab` are complete <sub>or this PR is not labeled `deploy:gitlab`</sub>
- [x] Background migrations for `anvilprod.gitlab` are complete <sub>or this PR is not labeled `deploy:gitlab`</sub>
- [x] PR is assigned to only the operator

### Operator (before pushing merge the commit)

- [x] Ran `_select dev.gitlab && make -C terraform/gitlab/runner` <sub>or this PR is not labeled `deploy:runner`</sub>
- [x] Ran `_select anvildev.gitlab && make -C terraform/gitlab/runner` <sub>or this PR is not labeled `deploy:runner`</sub>
- [x] Ran `_select anvilprod.gitlab && make -C terraform/gitlab/runner` <sub>or this PR is not labeled `deploy:runner`</sub>
- [x] Added `sandbox` label <sub>or PR is labeled `no sandbox`</sub>
- [x] Pushed PR branch to GitLab `dev` <sub>or PR is labeled `no sandbox`</sub>
- [x] Pushed PR branch to GitLab `anvildev` <sub>or PR is labeled `no sandbox`</sub>
- [x] Pushed PR branch to GitLab `anvilprod` <sub>or PR is labeled `no sandbox`</sub>
- [x] Build passes in `sandbox` deployment <sub>or PR is labeled `no sandbox`</sub>
- [x] Build passes in `anvilbox` deployment <sub>or PR is labeled `no sandbox`</sub>
- [x] Build passes in `hammerbox` deployment <sub>or PR is labeled `no sandbox`</sub>
- [x] Reviewed build logs for anomalies in `sandbox` deployment <sub>or PR is labeled `no sandbox`</sub>
- [x] Reviewed build logs for anomalies in `anvilbox` deployment <sub>or PR is labeled `no sandbox`</sub>
- [x] Reviewed build logs for anomalies in `hammerbox` deployment <sub>or PR is labeled `no sandbox`</sub>
- [x] Deleted unreferenced indices in `sandbox` <sub>or this PR does not remove catalogs or otherwise causes unreferenced indices in `dev`</sub>
- [x] Deleted unreferenced indices in `anvilbox` <sub>or this PR does not remove catalogs or otherwise causes unreferenced indices in `anvildev`</sub>
- [x] Deleted unreferenced indices in `hammerbox` <sub>or this PR does not remove catalogs or otherwise causes unreferenced indices in `anvilprod`</sub>
- [x] Started reindex in `sandbox` <sub>or this PR is not labeled `reindex:dev`</sub>
- [x] Started reindex in `anvilbox` <sub>or this PR is not labeled `reindex:anvildev`</sub>
- [x] Started reindex in `hammerbox` <sub>or this PR is not labeled `reindex:anvilprod`</sub>
- [x] Checked for failures in `sandbox` <sub>or this PR is not labeled `reindex:dev`</sub>
- [x] Checked for failures in `anvilbox` <sub>or this PR is not labeled `reindex:anvildev`</sub>
- [x] Checked for failures in `hammerbox` <sub>or this PR is not labeled `reindex:anvilprod`</sub>
- [x] The title of the merge commit starts with the title of this PR
- [x] Added PR # reference to merge commit title
- [x] Collected commit title tags in merge commit title <sub>but only included `p` if the PR is also labeled `partial`</sub>
- [x] Moved connected issues to Merged column in ZenHub
- [x] Pushed merge commit to GitHub

### Operator (chain shortening)

- [x] Changed the target branch of the blocked PR to `develop` <sub>or this PR is not labeled `base`</sub>
- [x] Removed the `chained` label from the blocked PR <sub>or this PR is not labeled `base`</sub>
- [x] Removed the blocking relationship from the blocked PR <sub>or this PR is not labeled `base`</sub>
- [x] Removed the `base` label from this PR <sub>or this PR is not labeled `base`</sub>

### Operator (after pushing the merge commit)

- [x] Pushed merge commit to GitLab `dev` <sub>or PR is labeled `no sandbox`</sub>
- [x] Pushed merge commit to GitLab `anvildev` <sub>or PR is labeled `no sandbox`</sub>
- [x] Pushed merge commit to GitLab `anvilprod` <sub>or PR is labeled `no sandbox`</sub>
- [x] Build passes on GitLab `dev`<sup>1</sup>
- [x] Reviewed build logs for anomalies on GitLab `dev`<sup>1</sup>
- [x] Build passes on GitLab `anvildev`<sup>1</sup>
- [x] Reviewed build logs for anomalies on GitLab `anvildev`<sup>1</sup>
- [x] Build passes on GitLab `anvilprod`<sup>1</sup>
- [x] Reviewed build logs for anomalies on GitLab `anvilprod`<sup>1</sup>
- [x] Ran `_select dev.shared && make -C terraform/shared apply` <sub>or this PR is not labeled `deploy:shared`</sub>
- [x] Ran `_select anvildev.shared && make -C terraform/shared apply` <sub>or this PR is not labeled `deploy:shared`</sub>
- [x] Ran `_select anvilprod.shared && make -C terraform/shared apply` <sub>or this PR is not labeled `deploy:shared`</sub>
- [x] Deleted PR branch from GitHub
- [x] Deleted PR branch from GitLab `dev`
- [x] Deleted PR branch from GitLab `anvildev`
- [x] Deleted PR branch from GitLab `anvilprod`

<sup>1</sup> When pushing the merge commit is skipped due to the PR being
labelled `no sandbox`, the next build triggered by a PR whose merge commit *is*
pushed determines this checklist item.

### Operator (reindex)

- [x] Deindexed all unreferenced catalogs in `dev` <sub>or this PR is neither labeled `reindex:partial` nor `reindex:dev`</sub>
- [x] Deindexed all unreferenced catalogs in `anvildev` <sub>or this PR is neither labeled `reindex:partial` nor `reindex:anvildev`</sub>
- [x] Deindexed all unreferenced catalogs in `anvilprod` <sub>or this PR is neither labeled `reindex:partial` nor `reindex:anvilprod`</sub>
- [x] Deindexed specific sources in `dev` <sub>or this PR is neither labeled `reindex:partial` nor `reindex:dev`</sub>
- [x] Deindexed specific sources in `anvildev` <sub>or this PR is neither labeled `reindex:partial` nor `reindex:anvildev`</sub>
- [x] Deindexed specific sources in `anvilprod` <sub>or this PR is neither labeled `reindex:partial` nor `reindex:anvilprod`</sub>
- [x] Indexed specific sources in `dev` <sub>or this PR is neither labeled `reindex:partial` nor `reindex:dev`</sub>
- [x] Indexed specific sources in `anvildev` <sub>or this PR is neither labeled `reindex:partial` nor `reindex:anvildev`</sub>
- [x] Indexed specific sources in `anvilprod` <sub>or this PR is neither labeled `reindex:partial` nor `reindex:anvilprod`</sub>
- [x] Started reindex in `dev` <sub>or this PR does not require reindexing `dev`</sub>
- [x] Started reindex in `anvildev` <sub>or this PR does not require reindexing `anvildev`</sub>
- [x] Started reindex in `anvilprod` <sub>or this PR does not require reindexing `anvilprod`</sub>
- [x] Checked for, triaged and possibly requeued messages in both fail queues in `dev` <sub>or this PR does not require reindexing `dev`</sub>
- [x] Checked for, triaged and possibly requeued messages in both fail queues in `anvildev` <sub>or this PR does not require reindexing `anvildev`</sub>
- [x] Checked for, triaged and possibly requeued messages in both fail queues in `anvilprod` <sub>or this PR does not require reindexing `anvilprod`</sub>
- [x] Emptied fail queues in `dev` <sub>or this PR does not require reindexing `dev`</sub>
- [x] Emptied fail queues in `anvildev` <sub>or this PR does not require reindexing `anvildev`</sub>
- [x] Emptied fail queues in `anvilprod` <sub>or this PR does not require reindexing `anvilprod`</sub>

### Operator

- [x] Propagated the `deploy:shared`, `deploy:gitlab`, `deploy:runner`, `reindex:partial` and `reindex:prod` labels to the next promotion PR <sub>or this PR carries none of these labels</sub>
- [x] Propagated any specific instructions related to the `deploy:shared`, `deploy:gitlab`, `deploy:runner`, `reindex:partial` and `reindex:prod` labels from the description of this PR to that of the next promotion PR <sub>or this PR carries none of these labels</sub>
- [x] PR is assigned to no one

## Shorthand for review comments

- `L` line is too long
- `W` line wrapping is wrong
- `Q` bad quotes
- `F` other formatting problem

### CODE REVIEW

**Comment by dsotirho-ucsc** [comment]
[6039-IT_2024-03-13.txt](https://github.com/DataBiosphere/azul/files/14593055/6039-IT_2024-03-13.txt)

**Review by achave11-ucsc — CHANGES_REQUESTED** [review]
From the issue,
>"Assignee to file partial PR that downgrades ClamAV. Ticket will then go to parked column until upstream fixes the segmentation fault."

PR lacks indications of partiality.

**Review by achave11-ucsc — DISMISSED** [review]
Approved ✅

File: environment.py [hunk 1]
```diff
@@ -284,7 +284,9 @@ def env() -> Mapping[str, Optional[str]]:
             # Updating any of the four images below additionally requires
             # redeploying the `gitlab` TF component.
             'clamav': {
-                'ref': 'docker.io/clamav/clamav:1.3.0-41'
+                # FIXME: https://github.com/DataBiosphere/azul/issues/6022
+                #        Keep CalmAV at 1.2.1 until 1.3.x failure is resolved
```
Comment (hannes-ucsc) [inline environment.py]:
```suggestion
                #        Keep ClamAV at 1.2.1 until 1.3.x failure is resolved
```

File: image_manifests.json [hunk 2]
```diff
@@ -64,9 +64,9 @@
         "id": "sha256:2fc698b738dd0feeab79fa175dbca3dc4cf59f7a766e16b85541327e6b3681fc",
         "platform": "linux/amd64"
     },
-    "docker.io/clamav/clamav:1.3.0-41": {
-        "digest": "sha256:0bb5d72ed47458c3333868c2e5835dd38f4268ceeba6e4212b3c57d809526755",
-        "id": "sha256:c9f71cb59795ba6a22385dd431cc8270a8215a1e50326fa293ce6895288ee08d",
+    "docker.io/clamav/clamav:1.2.1-27": {
+        "digest": "sha256:f51c38f31c6b8754191a040c9f9ffc7cb3e44c796b9a2700b3cfd33211651a6f",
+        "id": "sha256:b56c8c4aa940277a4dc17599b610ce55ab09d3d882fee0e3bb6ecabd2748a5ee",
```
Comment (hannes-ucsc) [inline image_manifests.json:69]: Only this hunk. Revert the other hunks in this file.

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]
It's not clear from the description or comments in the GH issue what the problem is. One of my comments is alluding to a segmentation fault but there is no actual error message mentioned in the issue that would corroborate that. Without this information it would be hard to file an upstream issue or look for an existing issue. It would also be good to know what file(s) cause ClamAV to crash.

**Comment by dsotirho-ucsc** [comment]
> It's not clear from the description or comments in the GH issue what the problem is. One of my comments is alluding to a segmentation fault but there is no actual error message mentioned in the issue that would corroborate that. Without this information it would be hard to file an upstream issue or look for an existing issue. It would also be good to know what file(s) cause ClamAV to crash.

Added verbose logs and reproduction to [the ticket](https://github.com/DataBiosphere/azul/issues/6022#issuecomment-2006172997) and linked to the comment in the ticket's description.

**Review by hannes-ucsc — APPROVED** [review]

File: environment.py [hunk 3]
```diff
@@ -284,7 +284,9 @@ def env() -> Mapping[str, Optional[str]]:
             # Updating any of the four images below additionally requires
             # redeploying the `gitlab` TF component.
```
Comment (hannes-ucsc) [inline environment.py:285]: NB

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]
PR labels are wrong. See checklist.

**Review by hannes-ucsc — APPROVED** [review]

**Comment by hannes-ucsc** [comment]
### Security design review

- [x] Security design review completed; this PR does **not** …
  - [x] … affect authentication; for example:
    - OAuth 2.0 with the application (API or Swagger UI)
    - Authentication of developers with Google Cloud APIs
    - Authentication of developers with AWS APIs
    - Authentication with a GitLab instance in the system
    - Password and 2FA authentication with GitHub
    - API access token authentication with GitHub
    - Authentication with Terra 
  - [x] … affect the permissions of internal users like access to
    - Cloud resources on AWS and GCP
    - GitLab repositories, projects and groups, administration
    - an EC2 instance via SSH
    - GitHub issues, pull requests, commits, commit statuses, wikis, repositories, organizations
  - [x] … affect the permissions of external users like access to
    - TDR snapshots
  - [x] … affect permissions of service or bot accounts
    - Cloud resources on AWS and GCP
  - [x] … affect audit logging in the system, like
    - adding, removing or changing a log message that represents an auditable event
    - changing the routing of log messages through the system
  - [x] … affect monitoring of the system
  - [x] … introduce a new software dependency like
    - Python packages on PYPI
    - Command-line utilities
    - Docker images
    - Terraform providers
  - [x] … add an interface that exposes sensitive or confidential data at the security boundary
  - [x] … affect the encryption of data at rest
  - [x] … require persistence of sensitive or confidential data that might require encryption at rest
  - [x] … require unencrypted transmission of data within the security boundary
  - [x] … affect the network security layer; for example by 
    - modifying, adding or removing firewall rules
    - modifying, adding or removing security groups
    - changing or adding a port a service, proxy or load balancer listens on
- [x] Documentation on any unchecked boxes is provided in comments below

**Pull request merged**